activemodel/lib/active_model/errors.rb

```diff
@@ -410,8 +412,9 @@ def full_message(attribute, message)
       defaults << :"errors.format"
       defaults << "%{attribute} %{message}"
 
-      attr_name = attribute.to_s.tr(".", "_").humanize
-      attr_name = @base.class.human_attribute_name(attribute, default: attr_name)
+      attr_name = attribute.tr(".", "_").humanize
+      attr_name = @base.class.human_attribute_name(attribute.to_sym, default: attr_name)
```

Comment:
I don't think it's necessary to call `to_sym`. `human_attribute_name` doesn't expect a string
https://github.com/rails/rails/blob/47eadb68bfcae1641b019e07e051aa39420685fb/activemodel/lib/active_model/translation.rb#L46

Reply:
Updated in a 2nd commit. It changes the expectations of 1 test, but there are other examples of `human_attribute_name` being called with a Strings, so I'd expect custom overrides of `human_attribute_name` already need to handle both string and symbol